scripts/lint-commits.sh

```diff
@@ -14,7 +14,14 @@ if [[ -n "${JENKINS_URL}" ]]; then
 
     # Lint PR title
     if [[ -n ${CHANGE_TITLE} ]]; then
+      titleLength=$(echo -n "${CHANGE_TITLE}" | wc -c)
+      titleResult=0
+      if [ "${titleLength}" -ge 65 ] ; then
```

Comment:
curious, any reason we are using number 65?

Reply:
IIRC, 72 is the default [header-max-length](https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md#header-max-length), so when squashing and merging then the title will add the PR number with the format ` (#xyz)`, that particular format is 7chars length.

So, I've decided to use `ge` to ensure if we get above 1000 PRs then the behaviour will keep as expected, to ensure the title takes into consideration the PR id.

This is the alternative until, https://github.com/watson/commitlint-config-squash-pr/pull/2 gets merged and released.